src/Resources/contao/classes/StyleManager.php

```diff
@@ -145,7 +145,7 @@ public static function updateClasses($varValue, $dc)
 
         if(self::isMultipleField($dc->field))
         {
-            $varValue = array($cssID[0], $varValue);
+            $varValue = serialize(array($cssID[0], $varValue));
```

Comment:
Via "edit multiple" we now get an error due to this addition, because the value is serialized multiple times. If we want to leave this addition at this point, we have to remove the serialization in the widget.

https://github.com/oveleon/contao-component-style-manager/blob/805a3a9bf7d46d33b75ac5ad603e5e0f8cbe522c/src/Resources/contao/widgets/ComponentStyleSelect.php#L305

Reply:
Okay, php8 was complaining about converting (or rather not converting) array to string later and I found this code at this point. I don't know what would be the best solution - you're more familiar with your code :grin: